src/status_im/ui/screens/wallet/main/views.cljs

```diff
@@ -38,7 +38,10 @@
   [rn/view {:style wallet-styles/toolbar-buttons-container}
    [vi/touchable-icon :icons/dots-vertical (merge {:color :white} wallet-styles/toolbar-icon) show-not-implemented!]
    [vi/touchable-icon :icons/transaction-history (merge {:color :white :style {:viewBox "-108 65.9 24 24"}} wallet-styles/toolbar-icon)
-    #(rf/dispatch [:navigate-to-modal :wallet-transactions])]])
+    #(rf/dispatch [:navigate-to-modal :wallet-transactions])]
+   (if false
+     ;; TODO placeholder for check on unsigned transactions count
```

Comment:
could you add your name please

Reply:
and this form looks weird, if it always false maybe better to comment it at all?

Reply:
That's a placeholder for future features not yet implemented. I will improve the comment.

Reply:
thanks